README.md

```diff
@@ -31,7 +31,7 @@ REQUIRE_VALID_CERTIFICATE=false
 Download the script and deploy the infrastructure:
 
 ```
-wget https://github.com/dynatrace-oss/dynatrace-aws-log-forwarder/releases/latest/download/dynatrace-aws-log-forwarder.zip && unzip -q dynatrace-aws-log-forwarder.zip && cd dynatrace-aws-log-forwarder && ./dynatrace-aws-logs.sh deploy --target-url $TARGET_URL --target-api-token $TARGET_API_TOKEN --require-valid-certificate $REQUIRE_VALID_CERTIFICATE
+wget -O dynatrace-aws-log-forwarder.zip https://github.com/dynatrace-oss/dynatrace-aws-log-forwarder/releases/latest/download/dynatrace-aws-log-forwarder.zip && unzip -qo dynatrace-aws-log-forwarder.zip && ./dynatrace-aws-logs.sh deploy --target-url $TARGET_URL --target-api-token $TARGET_API_TOKEN --require-valid-certificate $REQUIRE_VALID_CERTIFICATE
```

Comment:
wget -O is what one-agent one-liner also does, to make sure the zip get overwritten on each call of the wget, rather than create a dynatrace-aws-log-forwarder.zip.1, dynatrace-aws-log-forwarder.zip.2 that will not be used by the unzip

unzip -o allows overwritting the target files without asking about each one of them. I found it annoying to have to click y(es) four times, but we can talk about it. Excerpt from one-liner output without -o below:
![image](https://user-images.githubusercontent.com/50921490/119457613-27e5ef80-bd3c-11eb-9231-6b9cafb608fb.png)

Reply:
Good catch! I haven't checked the overwrite-on-unzip scenario and didn't notice that before

Reply:
Tested with one-liner from this PR with a slight alteration for custom release, works as planned:
```
#RELEASE="latest"
RELEASE="release-test-new-zip-and-oneliner"
GITHUB_RELEASE=https://github.com/dynatrace-oss/dynatrace-aws-log-forwarder/releases/download/$RELEASE/dynatrace-aws-log-forwarder.zip
wget -O dynatrace-aws-log-forwarder.zip $GITHUB_RELEASE && unzip -qo dynatrace-aws-log-forwarder.zip && ./dynatrace-aws-logs.sh deploy --target-url $TARGET_URL --target-api-token $TARGET_API_TOKEN --require-valid-certificate $REQUIRE_VALID_CERTIFICATE
```
![image](https://user-images.githubusercontent.com/50921490/119482035-b154ec00-bd53-11eb-8de0-d2593ba015d6.png)